### DISCUSSION TITLE
Export plot for publication.

### DISCUSSION THREAD

**Discussion by rk2802** [PoisonAlien/maftools#805] [Q&A]
Hello

How can we get the visualized plot in .pdf or .tif or any other format by using maftools, so that it can be used for publication?

**Comment by PoisonAlien** [comment]
Hi,
You can just call the `pdf` or `png` or `tiff` to save the plot. See here for an [example](https://bookdown.org/ndphillips/YaRrr/saving-plots-to-a-file-with-pdf-jpeg-and-png.html)


```r
pdf(file = "plot.pdf")
oncoplot(maf)
dev.off()
```

**Comment by rk2802** [comment]
Thank you so much. Its working

**Comment by FriederikeHanssen** [comment]
Hey! Is there an option to export to svg? I tried and it seems to be always be missing fonts among other things

**Reply by PoisonAlien** [reply]
I havent tried myself with the svg functions. Wouldn't a pdf be the same with a transparent background? or could you convert pdf to SVG with the ImageMagick `convert` command?

**Reply by FriederikeHanssen** [reply]
Of course. Thanks this worked well. I used inkscape (https://alpha.inkscape.org/vectors/www.inkscapeforum.com/viewtopic45f7-2.html?t=3688)
